pkg/cidata/cidata.TEMPLATE.d/boot/60-propagate-timezone.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -eux
+
+tz=$LIMA_TIMEZONE
```

Comment:
Not sure why you need to create an alias for the variable; why not use the full name below?

Reply:
Updated in [97c113c](https://github.com/lima-vm/lima/pull/1931/commits/97c113cde70a6db042accad70143046000a44da2), mainly because I forgot polish some debugging code.